binding/rust/build.rs

```diff
@@ -0,0 +1,62 @@
+/*
+    Copyright 2021-2021 Picovoice Inc.
```

Comment:
oh sorry, I meant just put "Copyright 2021 Picovoice Inc.". When it's the one year, we don't need to have a dash.